<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="totalGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>891</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>주차 기록 검색창</string>
   </property>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>28</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전화번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>차량번호</string>
    </property>
   </widget>
   <widget class="QComboBox" name="nameBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="carNumBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="phoneBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="detailGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>581</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>입차 일시</string>
     </property>
    </widget>
    <widget class="QComboBox" name="locationBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>주차 위치</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>차종</string>
     </property>
    </widget>
    <widget class="QComboBox" name="carKindBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RFID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="rfidBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>441</width>
       <height>26</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QDateTimeEdit" name="carOutDT_1">
      <property name="date">
       <date>
        <year>2024</year>
        <month>11</month>
        <day>10</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy. MM. dd. AP hh:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>부터</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="carOutDT_2">
      <property name="date">
       <date>
        <year>2024</year>
        <month>11</month>
        <day>15</day>
       </date>
      </property>
      <property name="time">
       <time>
        <hour>11</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="displayFormat">
       <string>yyyy. MM. dd. AP hh:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>까지</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>출차 일시</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>441</width>
       <height>26</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QDateTimeEdit" name="carInDT_1">
      <property name="date">
       <date>
        <year>2024</year>
        <month>11</month>
        <day>10</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy. MM. dd. AP hh:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>부터</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="carInDT_2">
      <property name="date">
       <date>
        <year>2024</year>
        <month>11</month>
        <day>15</day>
       </date>
      </property>
      <property name="time">
       <time>
        <hour>11</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="displayFormat">
       <string>yyyy. MM. dd. AP hh:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>까지</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMore">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>추가 정보</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>속성 초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="dbRecordWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>891</width>
     <height>551</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>RFID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>전화번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>차량번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>차종</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>주차위치</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>입차시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>출차시간</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnDBClear">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>910</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>주차 기록 DB 초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>910</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
